rpcd/playbooks/roles/kibana/tasks/kibana_post_install.yml

```diff
@@ -62,25 +96,13 @@
     - kibana-apache
     - kibana-post-install
 
-- name: Kibana Config
-  template:
-    src: "{{ item }}"
-    dest: "{{ kibana_root }}/{{ item }}"
-    owner: "{{ kibana_system_user }}"
-    group: "{{ kibana_system_group }}"
-  with_items:
-    - config.js
-  notify: Restart Apache
-  tags:
-    - kibana-post-install
-
 - name: Link Kibana Site
   file:
     state: "link"
     src: "/etc/apache2/sites-available/000-kibana.conf"
     dest: "/etc/apache2/sites-enabled/000-kibana.conf"
-    owner: "{{ kibana_system_user }}"
-    group: "{{ kibana_system_group }}"
```

Comment:
@d34dh0r53 So all of the installed apache2 files (/etc/apache2/*) are owned by root:root. So that template does not need to be owned by the kibana_system_user:kibana_system_group